charts/corda-dev/values.yaml

```diff
@@ -61,8 +64,8 @@ postgresql:
     # -- name of database to be created.
     database: cordacluster
     # -- name of the user to be created.
-    username: user
-    # -- name of the password of the user to be created - defaults to generated value
+    username: corda
```

Comment:
I'd leave this unchanged unless you have good reason to as there are places that assume that it is `user` e.g. https://github.com/corda/corda-runtime-os/blob/release/os/5.2/.ci/e2eTests/corda.yaml#L74